packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -118,7 +118,7 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
         role='tab'
         aria-controls={ id + '-panel' }
         aria-selected={ this.state.selectedKey === itemKey }>
-        { link.linkText }
+        <span className='ms-Pivot-text'>{ link.linkText }</span>
         { countText }
```

Comment:
`countText` is set in line 107 with a span, maybe the `Text` part of `countText` isn't perfect